crates/sui-bcs-json/sui_bcs_json_bindings.h

```diff
@@ -0,0 +1,74 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+/**
+ * Return 0 for success and 1 for failure
+ *
+ * Converts the JSON data into a BCS array.
+ * The result points to the address where the new BCS
+ * array is stored. Don't forget to deallocate the memory
+ * by calling the sui_bcs_json_free_array function.
+ *
+ * # Safety
+ * Unsafe function.
```

Comment:
I'm not sure what this `Safety` section is meant to convey.

Reply:
Clippy complains without a # Safety comment. I think I can add a ignore lint for it though, but that was my first thought to do to get rid of the issue.